linux/LinuxProcessList.c

```diff
@@ -1487,6 +1489,7 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
          } else {
             /* Copy from process structure in threads and reset if setting got disabled */
             proc->usesDeletedLib = (proc->isUserlandThread && parent) ? parent->usesDeletedLib : false;
+            lp->m_lrs = (proc->isUserlandThread && lparent) ? lparent->m_lrs : 0;
```

Comment:
I'm not sure it's really worth changing the function signature for one place where we need access to the casted subclass …

```suggestion
            lp->m_lrs = (proc->isUserlandThread && parent) ? ((LinuxProcess_t *)parent)->m_lrs : 0;
```
And skip renaming the signature of this function …

Reply:
Done.